packages/ember-cookies/src/test-support/index.js

```diff
@@ -0,0 +1 @@
+export { default as clearAllCookies } from '../clear-all-cookies';
```

Comment:
as this is actually test helper, by conventions it should come `ember-cookies/test-support` path.

@marcoow please let me know if you don't agree.

Reply:
Yes, if this is the convention let's change the way this needs to be imported as this is breaking anyway :+1

Reply:
done! import path outlined in the `README.md`